Tests/Sources/VLCAudioTest.swift

```diff
@@ -0,0 +1,118 @@
+/*****************************************************************************
+ * VLCAudioTest.swift
+ *****************************************************************************
+ * Copyright (C) 2018 Mike JS. Choi
+ * Copyright (C) 2018 VLC authors and VideoLAN
+ * $Id$
+ *
+ * Authors: Mike JS. Choi <mkchoi212 # icloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston MA 02110-1301, USA.
+ *****************************************************************************/
+
+import XCTest
+
+class VLCAudioTest: XCTestCase {
+    
+    let step = 6
+    let min = 0
+    let max = 200
+    
+    func testIsMuted() throws {
+        let player = VLCMediaPlayer()
+        let audio = try XCTAssertNotNilAndUnwrap(player.audio)
+        XCTAssertFalse(audio.isMuted)
+        
+        audio.isMuted = true
+        XCTAssertTrue(audio.isMuted)
+        
+        audio.isMuted = false
+        XCTAssertFalse(audio.isMuted)
+        
+        audio.setMute(true)
```

Comment:
huh, I wonder if we really need the setter and the property. @fkuehne ?

Reply:
I think this can be removed since this is a deprecated API

Reply:
Yes, this API will be removed in v4.